src/react.force.common.js

```diff
@@ -53,12 +53,8 @@ export const exec = (moduleIOSName, moduleAndroidName, moduleIOS, moduleAndroid,
             result => {
                 console.log(`${func} succeeded`);
                 if (successCB) {
-                    try {                        
-                        const resultParsed = result ? JSON.parse(result) : result;
-                        successCB(resultParsed);
```

Comment:
Old code: if successCB throws an error, it gets called a second time from the catch block!
We had the try / catch to catch parsing error.
I got rid of it, if we can't parse it then the native bridge code has a bug, and we want to realize it quickly.